CHANGELOG.md

```diff
@@ -1,5 +1,207 @@
+- Remove unused styles (#22083) ([ffd7195](https://github.com/facebook/react-native/commit/ffd7195) by [@vcalvello](https://github.com/vcalvello))
+- Fix `no-shadow` eslint warning & remove var (#22124) ([f8040ed](https://github.com/facebook/react-native/commit/f8040ed) by [@Tnarita0000](https://github.com/Tnarita0000))
+- Removed unnecessary code in Libraries/Text/Text.js (#22132) ([0d4f627](https://github.com/facebook/react-native/commit/0d4f627) by [@ifsnow](https://github.com/ifsnow))
+- Remove var in /Libralies/Experimental/IncrementalPresenter.js (#22144) ([cc90c20](https://github.com/facebook/react-native/commit/cc90c20) by [@soyanakagawa](https://github.com/soyanakagawa))
+- Remove var in Libraries/Utilities/deepFreezeAndThrowOnMutationInDev.js (#22126) ([0a39cda](https://github.com/facebook/react-native/commit/0a39cda) by [@nabettu](https://github.com/nabettu))
+- Remove unused loads from xplat. ([9b781bd](https://github.com/facebook/react-native/commit/9b781bd) by [@ttsugriy](https://github.com/ttsugriy))
+- Remove dynamic exception specification in RN MethodCall.h/cpp ([5b71408](https://github.com/facebook/react-native/commit/5b71408) by [@yiding](https://github.com/yiding))
+- NIT remove unnecessary cast in measure ([2dbe769](https://github.com/facebook/react-native/commit/2dbe769) by [@mdvacca](https://github.com/mdvacca))
+- Remove useless additionnal blur call (#22156) ([27cfba2](https://github.com/facebook/react-native/commit/27cfba2))
+- `YGNodeComputeFlexBasisForChildren`: remove output param ([8f283b9](https://github.com/facebook/react-native/commit/8f283b9) by [@davidaurelio](https://github.com/davidaurelio))
+
+#### Android specific
+
+- Remove AndroidManifest.xml from UIManager (#22044) ([7f79254](https://github.com/facebook/react-native/commit/7f79254) by [@radeno](https://github.com/radeno))
+- remove var in ReactAndroid/src/androidTest (#22136) ([0beb1cc](https://github.com/facebook/react-native/commit/0beb1cc) by [@nd-02110114](https://github.com/nd-02110114))
+- remove var in ReactAndroid/src/androidTest. (#22137) ([6f781d9](https://github.com/facebook/react-native/commit/6f781d9) by [@nd-02110114](https://github.com/nd-02110114))
+- Remove var in ReactAndroid/src/androidTest (#22135) ([9d13233](https://github.com/facebook/react-native/commit/9d13233) by [@nd-02110114](https://github.com/nd-02110114))
+- remove createReactClass from ToolbarAndroid/ToolbarAndroid.android.js (#21893) ([147c38a](https://github.com/facebook/react-native/commit/147c38a) by [@nd-02110114](https://github.com/nd-02110114))
+
+#### iOS specific
+
+### Fixed
+
+- Fix View/Text displayName (#21950) ([7a914fc](https://github.com/facebook/react-native/commit/7a914fc) by [@rajivshah3](https://github.com/rajivshah3))
+- Fix the lazily LaodedView to avoid weird naming issues ([cae2534](https://github.com/facebook/react-native/commit/cae2534) by [@spredolac](https://github.com/spredolac))
```

Comment:
```suggestion
- Fix the lazily LoadedView to avoid weird naming issues ([cae2534](https://github.com/facebook/react-native/commit/cae2534) by [@spredolac](https://github.com/spredolac))
```